releases/v20.1.0-alpha.20200210.md

```diff
@@ -0,0 +1,351 @@
+- We previously introduced a fix on the Admin UI to prevent non-admin users from executing queries. However, this inadvertently caused certain pages requiring table details not to display. This issue has now been resolved. [#44167][#44167]
+- Non-admin users can now use the [Jobs](../v20.1/admin-ui-jobs-page.html) page of the Admin UI to see their own jobs. [#44345][#44345]
+
+### Bug fixes
+
+- Previously, when vectorized execution engine was used with `vectorize=experimental_on`, CockroachDB could incorrectly report some values as NULL. This has now been fixed. [#43785][#43785]
+- When casting a string to bytes, octal escapes greater than `\377` will now generate an error, rather than silently wrapping around. [#43806][#43806]
+- Job can be running but shown in state pending. [#43814][#43814]
+- On Linux machines, we now respect the available memory limit as dictated by the cgroup limits which apply to the cockroach process. [#43137][#43137]
+- Previously, CockroachDB would return incorrect results for some aggregate functions when used as window functions with non-default window frame. This is now fixed. Note that MIN, MAX, SUM, AVG, and "pure" window functions (i.e., non-aggregates) were not affected. [#39308][#39308]
+- Previously, CockroachDB could return an internal error when running a query with a CAST operation (`:::`) in some cases when using the vectorized execution engine. This is now fixed. [#43857][#43857]
+- Previously, a query shutdown mechanism could fail to fully cleanup the infrastructure when the query was executed via the vectorized engine and the query plan contained wrapped row-by-row processors (in v19.2, this applies to Lookup joins and Index joins). This is now fixed. [#43579][#43579]
+- Fixed a bug introduced in v19.2 that would allow foreign keys to use a unique index on the referenced columns that indexed more columns than were included in the columns used in the FK constraint, which allows potentially violating uniqueness in the referenced columns themselves. [#43793][#43793]
+- Ensure that RESTORE cleanup is run exactly once. [#43933][#43933]
+- A benign error previously logged at the ERROR level is now logged at the INFO level behind a verbosity(2) flag. This error might have been observed as "context canceled: readerCtx in Inbox stream handler". [#44020][#44020]
+- A bug causing lost update transaction anomalies was fixed. [#42969][#42969]
+- Previously, an internal error could occur when a query with an aggregate function MIN or MAX was executed via the vectorized engine when the input column was either INT2 or INT4 type. This is now fixed. [#43985][#43985]
+- CDC is no longer susceptible to a bug where a resolved timestamp might be published before all events that precede it have been published in the presence of a Range merge. [#44035][#44035]
+- `cockroach zip` now emits the `goroutine` file in the proper sub-directory when the corresponding call fails with an error. [#44064][#44064]
```

Comment:
This appears to be a typo: there is no `cockroach zip` command, but there is `cockroach debug zip`.

This and the other mentions should probably link to https://www.cockroachlabs.com/docs/v19.2/cockroach-debug-zip.html